src/cryptenroll/cryptenroll.c

```diff
@@ -250,11 +257,35 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
 
                 case ARG_UNLOCK_KEYFILE:
+                        if (arg_unlock_type > UNLOCK_PASSWORD)
```

Comment:
`>` comparisons for enums are always weird, given they indicate ordering of the enum values was important. but that#s not really the case here. You might as well just to the more obvious `!=` comparison here...